lecture06.tex

```diff
@@ -14,31 +14,42 @@ \section{Lecture 6}
   Since $\Ocal$ is Noetherian, there exists a maximal witness $\afrak$ contradicting the assertion.
   Note that $\afrak$ is not prime, and thus we may find $x,y \in \Ocal$ such that $x \cdot y \in \afrak$ but $x,y \notin \afrak$.
   The ideals $\afrak + (x)$ and $\afrak + (y)$ are strictly larger than $\afrak$, so the maximality of $\afrak$ ensures that we can find a product of primes $\pfrak_{1} \cdots \pfrak_{k}$ contained in $\afrak + (x)$ and a product of primes $\qfrak_{1} \cdots \qfrak_{s}$ contained in $\afrak + (y)$.
-  The product $\pfrak_{1} \cdots \pfrak_{k} \cdot \qfrak_{1} \cdots \qfrak_{s}$ is then \todo{contained}{Add details.} in $\afrak$.
+  The product $\pfrak_{1} \cdots \pfrak_{k} \cdot \qfrak_{1} \cdots \qfrak_{s}$ is then contained in $\afrak$.
+  This product of ideals is generated by elements of the form
+  \[\prod_{i=1^k}(p_i+r_ix)\cdot\prod_{i=1}^s(q_i+t_iy)\]
+  With $p_i, q_i \in \afrak$, $r_i, t_i \in \Ocal$. 
+  Expanding, this product is observed to be of the form $a+cxy$ with $a \in \afrak$, $c \in \Ocal$.
+  This element is thus in $\afrak$, as $xy \in \afrak$.
 \end{proof}
 
 \begin{definition}
   For a fractional ideal $\afrak$ of $\Ocal$, we write
   \[ \afrak^{-1} := \{x \in K \ | \ x \cdot \afrak \subset \Ocal\}. \]
-  \todo{This is again a fractional ideal.}{Why?}
+  This is again a fractional ideal: it's clear that this is an $\Ocal$ module, so we must show it is finitely generated.
+  Let $x \in \afrak^{-1}$, and fix some generator of $\afrak$, say $\frac pq$.
```

Comment:
This seems to suggest that $\mathfrak{a}$ is generated by one element, $\frac pq$.

Reply:
True, and it doesn't even need to be a generator - that's just where my train of thought started